component/alert.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="/style.css" />
+  <link rel="stylesheet" href="/component/alert.css" />
+
+  <title>PineUI | Alert</title>
+</head>
+
+<body>
+  
+    <header class="header">
+      <figure class="logo-container">
+        <a href="../index.html">
+          <img src="/assets/logo.png" alt="logo" />
+        </a>
+      </figure>
+      <div class="header-links">
+        <ul class="lg-text">
+          <li><a href="/index.html">Home</a></li>
+         <li><a href="/component/alert.html">Docs</a></li>
+         <li><a href="https://github.com/prajapati-priyanka">Github</a></li>
+        </ul>
+      </div>
+    </header>
+    <main class="component-container wrapper">
+      <aside class="sidenav">
+        <h3 class="sidenav-heading">Components</h3>
+        <ul>
+          <li><a class=" active md-text" href="/component/alert.html">Alert</a></li>
```

Comment:
Empty space before `active` could be avoided.

Reply:
okay, I will do that.